vagrant/provisioning/roles/backup_data/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+- name: ensure Solr is stopped
+  become: yes
+  systemd:
+    name: solr
+    state: stopped
+
+- name: ensure Alfresco is stopped
+  become: yes
+  systemd:
+    name: alfresco
+    state: stopped
+
+- name: ensure Snowbound is stopped
+  become: yes
+  systemd:
+    name: snowbound
+    state: stopped    
+    
+- name: backup data folders 
+  become: yes
+  file:
+    path: "{{ root_folder }}/backups/data"
+    state: present
+
+- name: Solr data backuping
+  become: yes
+  copy:
+    src: "{{ root_folder }}/data/solr"
+    dest: "{{ root_folder }}/backups/data/solr-{{ ansible_date_time.iso8601 }}"
+    directory_mode: yes
+    owner: solr
+    group: solr
+    backup: yes
+
+- name: Alfresco data backuping
```

Comment:
We need to be clear here. This "backup" is not enough to restore Alfresco. I am not foo familiar with Alfresco, but I suppose a backup of the database would be required as well, and may of configuration files. A "backup" usually implies that you can restore a state from it.

I would suggest to rename this task "Backing up Alfresco files", to make it more clear what is being done here.